From 2fb48992f35e15400cfee9e5fbb01d3c68fdfd04 Mon Sep 17 00:00:00 2001
From: Simon Xue <xxm@rock-chips.com>
Date: Tue, 22 Oct 2019 08:19:22 +0800
Subject: [PATCH] ARM: dts: rockchip: rk312x: add
 rockchip,disable-device-link-resume for vop

vop iommu handled by vop driver, so ignore the iommu operation when vop
call pm_runtime_get_sync/pm_runtime_put_sync

Change-Id: I126661e56f16b1740793dcc49340518094eee514
Signed-off-by: Simon Xue <xxm@rock-chips.com>
---
 arch/arm/boot/dts/rk312x.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk312x.dtsi b/arch/arm/boot/dts/rk312x.dtsi
index 9e2d02d32c43..469ca5783adc 100644
--- a/arch/arm/boot/dts/rk312x.dtsi
+++ b/arch/arm/boot/dts/rk312x.dtsi
@@ -661,6 +661,7 @@
 		clock-names = "aclk", "iface";
 		power-domains = <&power RK3128_PD_VIO>;
 		#iommu-cells = <0>;
+		rockchip,disable-device-link-resume;
 		status = "disabled";
 	};
 
-- 
2.35.3

